lib/changed_rpm_files_helper.rb

```diff
@@ -33,11 +33,25 @@ def parse_rpm_changes_line(line)
       changes << "replaced"
       path.slice!(/ \(replaced\)$/)
     else
+      changes << "size" if rpm_changes[0] == "S"
       changes << "mode" if rpm_changes[1] == "M"
       changes << "md5" if rpm_changes[2] == "5"
+      changes << "device_number" if rpm_changes[3] == "D"
+      changes << "link_path" if rpm_changes[4] == "L"
       changes << "user" if rpm_changes[5] == "U"
       changes << "group" if rpm_changes[6] == "G"
+      changes << "time" if rpm_changes[7] == "T"
+      changes << "capabilities" if rpm_changes[8] == "P"
     end
+
+    if changes.empty?
+      changes << "other_rpm_change"
```

Comment:
Wouldn't `other_rpm_changes` be better? We don't know if it's only one change, no?